rmgpy/data/kinetics/family.py

```diff
@@ -2758,63 +2758,6 @@ def add_atom_labels_for_reaction(self, reaction, output_with_resonance=True):
             for species in reaction.reactants + reaction.products:
                 species.generate_resonance_structures()
 
-    def get_w0(self, rxn):
-        """
-        calculates the w0 for Blower Masel kinetics by calculating wf (total bond energy of bonds formed)
-        and wb (total bond energy of bonds broken) with w0 = (wf+wb)/2
-        """
-        mol = None
-        a_dict = {}
```

Comment:
I checked and these functions are essentially the same, but the one we are keeping in arrhenius.pyx has a few additional PEP-8 changes. Can you make these changes?

- `ADict` --> `a_dict`
- Opitionally, things like `bd2bde` --> `bd2_bde`. There are a couple of these variables that I different, but it is PEP-8 as is, so only change if you prefer it

Reply:
Done